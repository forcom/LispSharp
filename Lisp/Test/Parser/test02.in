(display "Hello World? Nice to meet you!")