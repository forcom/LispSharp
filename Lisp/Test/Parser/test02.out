((display "Hello World? Nice to meet you!"))

display : [Lisp.Symbol]
"Hello World? Nice to meet you!" : [System.String]
